subsys/bluetooth/host/hci_core.c

```diff
@@ -633,7 +628,7 @@ int bt_le_create_conn_legacy(const struct bt_conn *conn)
 int bt_le_create_conn(const struct bt_conn *conn)
 {
 	if (IS_ENABLED(CONFIG_BT_EXT_ADV) &&
-	    BT_FEAT_LE_EXT_ADV(bt_dev.le.features)) {
+		    printk("Ext create conn\n");
```

Comment:
Remove?

Reply:
My partial staging was wrong here, I removed the wrong line. Removed the printk, and added back the feature bit check.